src/bt-unix.c

```diff
@@ -19,30 +20,57 @@
 // ^ TODO: add ability to find differently named EV3's
 
 
+typedef struct {
+	int read_fd;
+	int write_fd;
+	struct termios old_read_mode;
+	struct termios old_write_mode;
+} bt_handle_t;
+
+int set_termios(struct termios *old, int fd) {
+	if (tcgetattr(fd, old) < 0) {
+		perror("cannot get tty attributes");
+		return -1;
+	}
+
+	struct termios new = *old;
+	new.c_iflag &= ~(BRKINT | ICRNL | IGNBRK | IGNCR | IGNPAR | INLCR | INPCK | ISTRIP | IXOFF | IXON | PARMRK);
+	new.c_oflag &= ~(OPOST);
+	new.c_cflag |= (CREAD | CS8);
```

Comment:
`CLOCAL` missing here. macOS doesn't set it by default, so I am wondering if this ever worked